Automattic/jetpack
# Retry/phpcs changed

Follow up from #13967 and #13931 

#### Changes proposed in this Pull Request:
* See #13931

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* n/a

#### Testing instructions:
* Build a production version and ensure vendor/automattic directories are not symlinks.
*

#### Proposed changelog entry for your changes:
* n/a
